### PR TITLE
Support advanced filtering

### PR DESCRIPTION
**Pull request by caothang53**
To address this issue https://github.com/iPriceGroup/ocapiplugin/issues/12

### CODE REVIEW

**Comment by caothang53** [comment]
@sandyandi I would like to use brackets format for specifying operators instead of colons because colons might be used in filter value. The format should be **?filter_field[operator]=filter_value**

File: README.md [hunk 1]
```diff
@@ -72,6 +95,21 @@ GET example.com/api/v1/blogs/1
 GET example.com/api/v1/tech/blogs/1
 ```
 
+To sort your results, use **sort** as a query parameter. **sort** parameter is format of `?sort[sort_order]=sort_field`. If `sort_order` is not specified, the default sort order `asc` will be used
```
Comment (sandyandi) [inline README.md]: This section of the documentation should only be in #11.

File: README.md [hunk 2]
```diff
@@ -46,6 +46,29 @@ GET example.com/api/v1/blogs
...
+GET example.com/api/v1/blogs/?id[gte]=2&id[lte]=200
+
+# Supporting operators:
+## 'gte': greater than and equal (>=)
+## 'gt': greater than (>)
+## 'lte': lesser than and equal (<=)
+## 'lt': lesser than (<)
+## 'eq': equal to (=) - used as default if not specified
+## 'neq': not equal to (<>)
+## 'like': like
```
Comment (sandyandi) [inline README.md]: How about using `contains` instead of `like` since `like` is a SQL term?

**Review by sandyandi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (caothang53) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (caothang53) [reply README.md]: Changed to `contains`

**Comment by zarina22** [comment]
Tested in QA environment. Its working as expected. QA passed

**Pull request merged**